src/api/resolve.ts

```diff
@@ -38,6 +39,8 @@ export function resolveVitestPackage(cwd: string, folder: vscode.WorkspaceFolder
 
 export function resolveVitestPackagePath(cwd: string, folder: vscode.WorkspaceFolder) {
   const customPackagePath = getConfig(folder).packagePath
+  if (existsSync(`${cwd}/.pnp.cjs`))
+    return null
```

Comment:
This will allow the extension to work when we have a yarn + pnp project inside a project running node (like the sample added to this project). If this is not done, we will fallback to `require.resolve()` below and the yarn project will be ignored and the extension does not work.